Mathlib/Topology/MetricSpace/Bounded.lean

```diff
@@ -172,6 +172,17 @@ theorem cobounded_le_cocompact : cobounded α ≤ cocompact α :=
 #align comap_dist_right_at_top_le_cocompact Metric.cobounded_le_cocompactₓ
 #align comap_dist_left_at_top_le_cocompact Metric.cobounded_le_cocompactₓ
 
+theorem _root_.Bornology.IsCobounded.closedBall_compl_subset {s : Set α} (hs : IsCobounded s)
```

Comment:
Could you turn this into an iff (and register the current implication as an alias)?